sphinxcontrib/confluencebuilder/translator.py

```diff
@@ -91,14 +91,25 @@ def depart_document(self, node):
 
         # prepend header (if any)
         if self.builder.config.confluence_header_file is not None:
+            header = ''
```

Comment:
I assume this was suppose to be `header_template_data`?

You'd also want to make sure the same changes would apply to `footer_template_data`.

Reply:
yup - fixed in both cases. Thanks!